src/api/test-helpers/cover-period.ts

```diff
@@ -0,0 +1,35 @@
+import { COVER_PERIOD } from '../constants';
+import { CoverPeriod, TestHelperCreate } from '../types';
+
+const coverPeriodQuery = 'id value valueId';
+
+/**
+ * create coverPeriod test helper
+ * Create an coverPeriod with mock data
+ * @param {Object} KeystoneJS context API
+ * @returns {Object} Created coverPeriod
+ */
+const create = async ({ context }: TestHelperCreate) => {
+  try {
+    console.info('Creating an coverPeriod (test helpers)');
+
+    const coverPeriod = (await context.query.CoverPeriod.createOne({
+      data: {
+        valueId: COVER_PERIOD.LESS_THAN_2_YEARS.DB_ID,
+        value: 'Mock value',
+      },
+      query: coverPeriodQuery,
+    })) as CoverPeriod;
+
+    return coverPeriod;
+  } catch (err) {
+    console.error(err);
```

Comment:
Can we add a message to `console.error` please with test `console.error('Error when creating cover period %s', error);`.

Reply:
Can we have this as tech debt? Can do after merging this PR. There's several test helpers we could do this for, would like to do in one go. Otherwise happy to do now. These are purely used for API unit tests.

Reply:
Sounds good.